### DISCUSSION TITLE
are  these  cool logos have copy right ????

### DISCUSSION THREAD

**Discussion by dreaminbb** [fastfetch-cli/fastfetch#1433] [Q&A]
hollo sir.
I try to use  mac logo as my blog  posts of hero image .
If it ok to use tell me the terms.
I love this logo .

**Comment by CarterLi** [comment]
I dont have the cooyright. All copyrights belong to their original authors.

**Comment by dreaminbb** [comment]
> I dont have the cooyright. All copyrights belong to their original authors.

Thanks for replying

Could you tell me how to check authors ?
